pages/app/controllers/refinery/admin/pages_dialogs_controller.rb

```diff
@@ -28,8 +28,8 @@ def link_to
         end
 
         # web address link
-        @web_address_text = "http://"
-        @web_address_text = params[:current_link] if params[:current_link].to_s =~ /^http:\/\//
+        @web_address_text = "https://"
+        @web_address_text = params[:current_link] if params[:current_link].to_s =~ /^https:\/\//
```

Comment:
shouldn't the `s` be optional? Like `/^https?:\/\//`

Reply:
right